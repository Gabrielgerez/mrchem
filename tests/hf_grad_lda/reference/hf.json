{
  "input": {
    "constants": {
      "N_a": 6.02214076e+23,
      "angstrom2bohrs": 1.8897261246257702,
      "boltzmann_constant": 1.380649e-23,
      "dipmom_au2debye": 2.5417464739297717,
      "e0": 8.8541878128e-12,
      "electron_g_factor": -2.00231930436256,
      "elementary_charge": 1.602176634e-19,
      "fine_structure_constant": 0.0072973525693,
      "hartree2ev": 27.211386245988,
      "hartree2kcalmol": 627.5094740630558,
      "hartree2kjmol": 2625.4996394798254,
      "hartree2simagnetizability": 78.9451185,
      "hartree2wavenumbers": 219474.6313632,
      "light_speed": 137.035999084,
      "meter2bohr": 18897261246.2577
    },
    "geom_opt": {
      "init_step_size": -0.5,
      "max_force_component": 0.005,
      "max_history_length": 10,
      "max_iter": 100,
      "minimal_step_size": 0.01,
      "run": false,
      "subspace_tolerance": 0.001,
      "use_previous_guess": false
    },
    "molecule": {
      "charge": 0,
      "coords": [
        {
          "atom": "f",
          "r_rms": 5.2927138943e-05,
          "xyz": [
            -2.25797417086055,
            -0.462275046496363,
            0.6442768685488515
          ]
        },
        {
          "atom": "h",
          "r_rms": 2.6569547399e-05,
          "xyz": [
            -3.1605765564318227,
            -0.7465980517921813,
            2.0925066015625777
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_per_node": -1,
      "bank_size": -1,
      "numerically_exact": true,
      "omp_threads": -1,
      "shared_memory_size": 10000,
      "use_omp_num_threads": false
    },
    "mra": {
      "basis_order": 5,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -6
    },
    "printer": {
      "file_name": "hf",
      "print_constants": false,
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "nuclear_model": "point_like",
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "svwn5"
              }
            ],
            "spin": false
          }
        }
      },
      "initial_guess": {
        "environment": "None",
        "external_field": "None",
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT (SVWN5)",
        "prec": 0.001,
        "relativity": "None",
        "restricted": true,
        "rotate": true,
        "screen": 12.0,
        "type": "core",
        "zeta": 2
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        },
        "geometric_derivative": {
          "geom-1": {
            "method": "hellmann_feynman",
            "operator": "h_nuc_grad",
            "precision": 0.001,
            "radius_factor": 0.5,
            "smoothing": 0.001,
            "surface_integral_precision": "medium"
          }
        }
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        -2.3034482007428676,
        -0.47659952984111875,
        0.7172401613655945
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 1.7062875014362446,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -0.887789451985288,
            -0.28714890143962,
            1.428563066477369
          ],
          "vector_el": [
            22.59455464218896,
            4.619924568820024,
            -6.4624353520247935
          ],
          "vector_nuc": [
            -23.48234409417425,
            -4.907073470259644,
            7.890998418502162
          ]
        }
      },
      "geometric_derivative": {
        "geom-1": {
          "electronic": [
            -0.09496103696802138,
            -3.4132265395264794,
            0.7938249209911681,
            -1.7368175804110166,
            -0.5507594994913632,
            2.7907795847178725
          ],
          "electronic_norm": 4.837107920879934,
          "nuclear": [
            -1.5689195606619417,
            -0.49421531749272585,
            2.5173387449218514,
            1.5689195606619417,
            0.49421531749272585,
            -2.5173387449218514
          ],
          "nuclear_norm": 4.252705427032731,
          "total": [
            -1.663880597629963,
            -3.9074418570192053,
            3.3111636659130195,
            -0.16789801974907492,
            -0.05654418199863731,
            0.2734408397960211
          ],
          "total_norm": 5.395049787911099
        }
      },
      "geometry": [
        {
          "symbol": "F",
          "xyz": [
            -2.25797417086055,
            -0.462275046496363,
            0.6442768685488515
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            -3.1605765564318227,
            -0.7465980517921813,
            2.0925066015625777
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -15.804626535069248,
          4.315120517981482,
          4.7428340772062,
          4.7724651360789485,
          5.076921041906711
        ],
        "occupation": [
          2.0,
          2.0,
          2.0,
          2.0,
          2.0
        ],
        "spin": [
          "p",
          "p",
          "p",
          "p",
          "p"
        ],
        "sum_occupied": 6.205428476208182
      },
      "scf_energy": {
        "E_ee": 94.63986905852005,
        "E_eext": 0.0,
        "E_el": -83.52282286402836,
        "E_en": -329.66039196257634,
        "E_kin": 166.83783976049733,
        "E_next": 0.0,
        "E_nn": 5.2023121410189415,
        "E_nuc": 5.2023121410189415,
        "E_tot": -78.32051072300942,
        "E_x": 0.0,
        "E_xc": -15.340139720469399,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 1,
      "routine": "mrchem.x",
      "total_cores": 1,
      "version": "1.2.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 94.63986905852005,
        "E_eext": 0.0,
        "E_el": -83.52282286402836,
        "E_en": -329.66039196257634,
        "E_kin": 166.83783976049733,
        "E_next": 0.0,
        "E_nn": 5.2023121410189415,
        "E_nuc": 5.2023121410189415,
        "E_tot": -78.32051072300942,
        "E_x": 0.0,
        "E_xc": -15.340139720469399,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
