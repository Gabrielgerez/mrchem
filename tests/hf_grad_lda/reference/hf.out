                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.2.0-alpha                              ***
***                                                                     ***
***         Git branch         text                                     ***
***         Git commit hash    12aee4301501e5cb99d9-dirty               ***
***         Git commit author  gitpeterwind                             ***
***         Git commit date    Thu Jan 2 09:29:25 2025 +0100            ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               1
 Total cores             :                                               1
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.6.0-alpha
 Git branch            : merge
 Git commit hash       : cba8256377a39fbe1bdd
 Git commit author     : gitpeterwind
 Git commit date       : Thu Feb 20 10:17:52 2025 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : NONE

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   64.00000    64.00000    64.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -64.00000   -64.00000   -64.00000 ]
 upper bounds          : [   64.00000    64.00000    64.00000 ]
 total length          : [  128.00000   128.00000   128.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.257974       -0.462275        0.644277
    1       H            :       -3.160577       -0.746598        2.092507
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize Core Hamiltonian matrix
 Precision               : 1.00000e-03
 Restricted              : True
 AO basis                : Hydrogenic orbitals
 Zeta quality            : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               5
 Beta electrons          :                                               5
 Total electrons         :                                              10
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.999841721759e-01
    1    2    p          :                              9.999998855685e-01
    2    2    p          :                              9.999999997972e-01
    3    2    p          :                              9.999741719440e-01
    4    2    p          :                              9.999890982281e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (SVWN5)
 Relativity              : None
 Environment             : None
 External fields         : None
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                166.837839760497
 E-N energy              :            (au)               -329.660391962576
 Coulomb energy          :            (au)                 94.639869058520
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -15.340139720469
 N-N energy              :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -83.522822864028
 Nuclear energy          :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.832051072301e+01
                         :      (kcal/mol)             -4.914686249215e+04
                         :        (kJ/mol)             -2.056304726671e+05
                         :            (eV)             -2.131209668267e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -15.804626535069
    1    2    p          :            (au)                  4.315120517981
    2    2    p          :            (au)                  4.742834077206
    3    2    p          :            (au)                  4.772465136079
    4    2    p          :            (au)                  5.076921041907
---------------------------------------------------------------------------
 Sum occupied            :            (au)                  6.205428476208
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.257974       -0.462275        0.644277
    1       H            :       -3.160577       -0.746598        2.092507
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                166.837839760497
 E-N energy              :            (au)               -329.660391962576
 Coulomb energy          :            (au)                 94.639869058520
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -15.340139720469
 N-N energy              :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -83.522822864028
 Nuclear energy          :            (au)                  5.202312141019
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.832051072301e+01
                         :      (kcal/mol)             -4.914686249215e+04
                         :        (kJ/mol)             -2.056304726671e+05
                         :            (eV)             -2.131209668267e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -15.804626535069
    1    2    p          :            (au)                  4.315120517981
    2    2    p          :            (au)                  4.742834077206
    3    2    p          :            (au)                  4.772465136079
    4    2    p          :            (au)                  5.076921041907
---------------------------------------------------------------------------
 Sum occupied            :            (au)                  6.205428476208
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       22.594555        4.619925       -6.462435
 Magnitude               :            (au)                       23.950379
                         :         (Debye)                       60.875792
---------------------------------------------------------------------------
 Nuclear vector          :      -23.482344       -4.907073        7.890998
 Magnitude               :            (au)                       25.254063
                         :         (Debye)                       64.189426
---------------------------------------------------------------------------
 Total vector            :       -0.887789       -0.287149        1.428563
 Magnitude               :            (au)                        1.706288
                         :         (Debye)                        4.336950
===========================================================================


===========================================================================
                       Geometric Derivative (geom-1)
---------------------------------------------------------------------------
 Total                   :       -1.663881       -3.907442        3.311164
                         :       -0.167898       -0.056544        0.273441
 Norm                    :            (au)                        5.395050
---------------------------------------------------------------------------
 Nuclear                 :       -1.568920       -0.494215        2.517339
                         :        1.568920        0.494215       -2.517339
 Norm                    :            (au)                        4.252705
---------------------------------------------------------------------------
 Electronic              :       -0.094961       -3.413227        0.793825
                         :       -1.736818       -0.550759        2.790780
 Norm                    :            (au)                        4.837108
---------------------------------------------------------------------------
 Est. var. of force error:            (au)                      2.3064e+00
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m  6s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
